api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/BasicValidationVTwo.java

```diff
@@ -174,7 +180,7 @@ public String validate(String validationType, String value){
         // This case may be needed for validation of other field types.
         if( "nonempty".equalsIgnoreCase(validationType)){
             if( isEmpty(value) )
-                return REQUIRED_FIELD_EMPTY_MSG;
+                return i18n.text("required_field_empty_msg");
```

Comment:
Should probably use the constant inplace of this String.
https://github.com/vivo-project/Vitro/pull/179/files#diff-c43254e24a1ff63bb1587b9707c50860R31